### ISSUE TITLE
How to cite the WASI spec?

### ISSUE THREAD

**Issue by jendrikw** [WebAssembly/WASI#260]
A similar discussion happened in https://github.com/WebAssembly/design/issues/1329.

I suggest
```bibtex
@report{WASI,
  title = {{WebAssembly} System Interface},
  author = {{WebAssembly Community Group}},
  date = {2020-02-29},
  url = {https://github.com/WebAssembly/WASI/blob/d8b286c697364d8bc4daf1820b25a9159de364a3/phases/snapshot/docs.md},
  langid = {american}
}
```

**Comment by sunfishcode** [comment]
I'm not a BibTeX expert, but that looks reasonable to me. Is anyone interested in creating a PR to add such a file to the repository?

**Comment by linclark** [comment]
I think that this issue should be fixed with #356. 

If there's still any work that needs to happen, please re-open this issue or file a new one.
